Sources/BraintreePayPal/BTPayPalDriver.m

```diff
@@ -147,88 +143,10 @@ - (void)requestPayPalCheckout:(BTPayPalRequest *)request
             return;
         }
 
-        NSMutableDictionary *parameters = [NSMutableDictionary dictionary];
```

Comment:
This chunk of code for creating the parameters dictionary now lives in `BTPayPalRequest`.